avaframe/out1Peak/outPlotAllPeak.py

```diff
@@ -110,15 +110,20 @@ def plotAllPeakFields(avaDir, cfgFLAGS, modName, demData=''):
             rowsMaxPlot = (rowsMax+1)*cellSize
             colsMinPlot = colsMin*cellSize
             colsMaxPlot = (colsMax+1)*cellSize
-            im0 = ax.imshow(demConstrained, cmap='Greys', extent=[colsMinPlot, colsMaxPlot, rowsMinPlot, rowsMaxPlot], origin='lower', aspect='equal')
-            im1 = ax.imshow(data, cmap=cmap, norm=norm, extent=[colsMinPlot, colsMaxPlot, rowsMinPlot, rowsMaxPlot], origin='lower', aspect='equal')
+
+            # add DEM hillshade with contour lines
+            ls, CS = pU.addHillShadeContours(ax, demConstrained, colsMinPlot, rowsMinPlot, cellSize,
+                extent=[colsMinPlot, colsMaxPlot, rowsMinPlot, rowsMaxPlot])
```

Comment:
feels wierd to pass twice the colsMinPlot, rowsMinPlot arguments. is 'extent' really optional? dont we always need it here?

Reply:
If (!) this is really needed, then make an `extent` variable which you can also then use in line 119